msal/src/Microsoft.Identity.Client/AccountId.cs

```diff
@@ -42,23 +42,30 @@ namespace Microsoft.Identity.Client
     public class AccountId
     {
         /// <summary>
-        /// An identifier for an account in a specific tenant
+        /// Unique identifier for the account
         /// </summary>
+        /// <remarks>
+        /// For Azure AD, Identifier is the concatenation of <see cref="ObjectId"/>and <see cref="TenantId"/>separated by a dot. 
+        /// Contrary to what was happening in ADAL.NET, these two segments are no longer base64 encoded.
+        /// </remarks>
         public string Identifier { get; }
 
         /// <summary>
-        /// The object id associated with the user that owns the account
+        /// For Azure AD, a string representation for a Guid which is the Object ID of the user owning the account in the tenant
```

Comment:
a string representation of a Guid ... maybe rephrase as "a guid", people know what this is.

Reply:
Not sure @bgavrilMS . A Guid is a type (System.Guid). Here this is its string representation?